networkmanager/src/main/java/com/dezen/riccardo/networkmanager/NetworkDictionary.java

```diff
@@ -9,7 +9,10 @@
 import java.util.List;
 
 /**
- * Class implementing Dictionary. Conceived as a double dictionary on SMSPeer and StringResource.
+ * @deprecated following the 14/11/19 lesson this class is being replaced by a similar class not
+ * involving relationships between Peers and Resources.
+ * Class implementing Dictionary. Conceived as a double dictionary on SMSPeer and StringResource,
+ * also allowing Peer ownership of Resources.
  * Due to trouble with testing android class. Lists have been used.
```

Comment:
Quali problemi?

Reply:
La classe ArraySet non può venire testata con gli UnitTest poichè facendo parte di un pacchetto della libreria Android ha bisogno dei mock test, tuttavia il problema non viene segnalato in compilazione se un suo esemplare è contenuto in un'altra classe, come Dictionary. Una volta identificato il problema l'implementazione era già proseguita in altro modo.